geaflow/geaflow-dsl/geaflow-dsl-plan/src/main/java/com/antgroup/geaflow/dsl/udf/table/string/SplitEx.java

```diff
@@ -34,4 +35,18 @@ public String eval(String str, String separator, Integer index) {
         }
         return values[index];
     }
+
+    public BinaryString eval(BinaryString str, BinaryString separator, Integer index) {
+        if ((str == null) || separator == null || index == null) {
+            return null;
```

Comment:
Special handling might be required here for cases where the separator is an empty string. Such a situation does not equate to null, but it may still not meet the expected criteria.

Reply:
sure, will fix it.